frontend/package.json

```diff
@@ -36,7 +37,7 @@
       ".(ts|tsx|js|jsx)": "./node_modules/ts-jest/preprocessor.js"
     },
     "transformIgnorePatterns": [
-      "<rootDir>/node_modules/(?!lodash-es/.*)"
+      "<rootDir>/node_modules/(?!lodash-es|@console)"
```

Comment:
To clarify, this change tells Jest to transform (basically "un-ignore") any `@console` module paths, in addition to existing `lodash-es` module paths.

More specifically, without this change, following module dependency chain:

```
__tests__/components/resource-pages.spec.tsx
public/components/resource-pages.ts
public/plugins.ts
```

will throw syntax errors (`export * from` statements) since `public/plugins.ts` imports from `@console/plugin-sdk` and _Jest doesn't transform stuff in `node_modules` by default._

This is also the reason why all Console monorepo packages should use the `@console` scope in their name.